custom_types.json

```diff
@@ -203,5 +203,18 @@
     "token_execution_exector_account_id": "u64",
     "token_execution_started_block": "Moment",
     "token_execution_ended_block": "Moment"
-  }
+  },
+  "ExchangeRateIndex": "u64",
+  "ExchangeRateConfig": {
+    "hbtc": "u64",
+    "dot": "u64",
+    "iota": "u64",
+    "fil": "u64",
+    "decimals_after_point": "u32"
```

Comment:
i think we'll need to use fixed point arithmetic for this https://substrate.dev/recipes/fixed-point.html